sdk/vision/imageAnalysis/samples/typescript/caption.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import createImageAnalysisClient, { ImageAnalysisClient, ImageAnalysisResultOutput } from '@azure/imageAnalysis';
+import { AzureKeyCredential } from '@azure/core-auth';
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint: string = process.env['VISION_ENDPOINT'] || '<your_endpoint>';
+const key: string = process.env['VISION_KEY'] || '<your_key>';
+const credential = new AzureKeyCredential(key);
+
+const client: ImageAnalysisClient = createImageAnalysisClient(endpoint, credential);
+
+const features: string[] = [
+  'Caption'
+];
+
+const imageUrl: string = 'https://aka.ms/azai/vision/image-analysis-sample.jpg';
+
+client.path('/imageanalysis:analyze').post({
+  body: { url: imageUrl },
+  queryParameters: { features: features },
+  contentType: 'application/json'
+}).then(result => {
```

Comment:
let's use async/await instead of then, await is more modern JS and easier to read and maintain

Reply:
This applies to other samples using then instead of async/await